src/backend/prisma/schema.prisma

```diff
@@ -61,6 +61,17 @@ model User {
   projectLead             WBS_Element[]    @relation(name: "projectLead")
   projectManager          WBS_Element[]    @relation(name: "projectManager")
   Session                 Session[]
+  teams                   Team[]           @relation(name: "teamsAsMember")     
+  teamAsLead              Team?            @relation(name: "teamAsLead")
+}
+
+model Team {
+  teamId         Int            @id @default(autoincrement())
+  teamName       String
+  leaderId       Int
+  leader         User           @relation(name: "teamAsLead", fields:[leaderId], references: [userId]) 
+  projects       Project[]      
+  members        User[]         @relation(name: "teamsAsMember")
```

Comment:
Should ask stakeholders about this but other things we could add:
- weekly meeting info: day of week, time, location, Zoom link
- intro / onboarding / team charter doc link: all business sub-teams have a charter doc, some have an onboarding doc, and I've heard some engineering teams have an intro / onboarding doc

Reply:
true true